<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-HN0JOUI" failures="1" tests="2" name="Test(failed)" time="0.102" errors="0" timestamp="2022-09-11T12:16:38 IST">
  <testcase classname="com.testng.TestNGRetryFailedDemo" name="test2" time="0.014"/>
  <testcase classname="com.testng.TestNGRetryFailedDemo" name="test3" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testng.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="com.testng.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Test(failed) -->
